Add Azure example tab to for_each meta-argument page (#2530)
hashicorp/web-unified-docs: content/terraform/v1.14.x/docs/language/meta-arguments/for_each.mdx

}
```

<Tabs>

> <Tab heading="aws">

Reviewer: ```suggestion

<Tab heading="AWS">
```

  name     = each.key
}
```
</Tab>

<Tab heading="Azure">

> The following example creates multiple Azure resource groups by iterating over a set of environment names with the `for_each` meta-argument.

Reviewer: ```suggestion
The following example iterates over a set of environment names with the `for_each` meta-argument to create multiple Azure resource groups.
```

```hcl
resource "azurerm_resource_group" "rg" {
 for_each = toset(["dev", "test", "staging", "prod"])
 name     = "rg-${each.key}"
 location = "eastus"
}

> ```

Reviewer: ````suggestion
```

````

  name     = each.key
}
```
</Tab>

<Tab heading="Azure">

> The following example iterates over a set of environment names with the `for_each` meta-argument to create multiple Azure resource groups.

Reviewer: ```suggestion

The following example iterates over a set of environment names with the `for_each` meta-argument to create multiple Azure resource groups.
```
